server/import.go

```diff
@@ -89,6 +115,38 @@ func createPublic(private tpm2.Private, hashAlg tpm2.Algorithm, pcrs *tpmpb.Pcrs
 			Unique: publicHash.Sum(nil),
 		},
 	}
+	setPublicAuth(&public, pcrs)
```

Comment:
Can we just call this in the `createImportBlobHelper`? That way we won't forget it as part of other import types.

Reply:
We can also then avoid needing to pass around `pcrs` as much